edx_exams/apps/api/v1/tests/test_views.py

```diff
@@ -256,3 +256,34 @@ def test_exams_config(self, other_course_id, exam_type, expect_none_provider):
         self.assertEqual(new_exam.due_date, pytz.utc.localize(datetime.fromisoformat('2025-07-01 00:00:00')))
         self.assertEqual(new_exam.hide_after_due, False)
         self.assertEqual(new_exam.is_active, True)
+
+
+@ddt.ddt
+class ProctoringProvidersViewTest(ExamsAPITestCase):
+    """
+    Tests ProctoringProvidersView
+    """
+
+    def get_response(self):
+        """
+        Helper function to make a get request
+        """
+        url = reverse("api:v1:proctoring-providers-list")
+        response = self.client.get(url)
+        return response
+
+    def test_proctoring_providers_list(self):
+
+        test_provider2 = ProctoringProvider.objects.create(
+            name='test_provider2',
+            verbose_name='testing_provider2',
+            lti_configuration_id='223456789'
+        )
+
+        response = self.get_response()
+        proctoring_providers_list = ProctoringProvider.objects.all()
+
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(len(proctoring_providers_list), 2)
+        self.assertIn(self.test_provider, proctoring_providers_list)
+        self.assertIn(test_provider2, proctoring_providers_list)
```

Comment:
these providers list tests aren't enough - they are testing that the django object layer works, which we know and don't need to test

if you want to validate that the providers come out in the response, that will require reparsing the response somewhat

Reply:
hm ok, would this be like checking the lti_config_id of the providers?

Reply:
you just need to get what you are testing from the response, not from a separately queried list

Reply:
You could hardcode exactly what you are testing. For example,

```expected_data = [{"name": "provider", "verbose_name": "verbose_provider", ...}, ...]`

You can then use assertEqual. Although, you may run into issues with the order of the list. But that's a solvable problem.